src/Servant/API/ContentTypes/Avro.hs

```diff
@@ -1 +1,28 @@
-module Servant.API.ContentTypes.Avro where
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+
+module Servant.API.ContentTypes.Avro (Avro) where
+
+import           Data.Avro                (FromAvro, Result (..), ToAvro,
+                                           decode, encode)
+import           Data.Proxy               (Proxy (..))
+import           Data.Typeable            (Typeable)
+import           Servant.API.ContentTypes
+
+-- The Avro encoding type for Servant
```

Comment:
If this is going to be published to Hackage, you may want to add Haddock markup here so that it will render docs appropriately.

Reply:
Durr, right. So
```-- | Foo```
instead of
```-- Foo```?

Reply:
👍